script/openqa-powermanagement.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3
+
+# TODO:
+#  * Add return values checks to avoid crashes/traces (no connection to openQA server, etc.)
+#  * The host name of machines may not be unique
+
+import configparser
+import argparse
+import json
+import requests
+import subprocess
+
+machine_list_idle = []
+machine_list_offline = []
+machine_list_busy= []
+machines_to_power_on = []
+
+jobs_worker_classes = []
+
+config_file = "/etc/openqa/openqa.ini"
+config = configparser.ConfigParser()
+config.read(config_file)
+
+openqa_server = "http://localhost"
+
+# Manage cmdline options
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--config')
+    parser.add_argument('--host')
+    parser.add_argument('--osd', action='store_true')
+    parser.add_argument('--o3', action='store_true')
+    args = parser.parse_args()
+    if args.config is not None and len(args.config):
+      config_file = args.config
+    if args.host is not None and len(args.host):
+      openqa_server = args.host
+    elif args.osd:
+      openqa_server = "http://openqa.suse.de"
```

Comment:
Using HTTPs here would like not hurt as well:
```suggestion
      openqa_server = "https://openqa.suse.de"
```

Reply:
`openqa-cli` uses http:// so I did the same, but if httpS works, I will update.

Reply:
https will work, thanks for updating